muxrpc/handlers/gossip/ping.go

```diff
@@ -19,7 +19,7 @@ import (
 //
 func Ping(ctx context.Context, req *muxrpc.Request, peerSrc *muxrpc.ByteSource, peerSnk *muxrpc.ByteSink) error {
 	type arg struct {
-		Timeout int
+		Timeout int // not actually a "timeout" but the delay between two pings
```

Comment:
why not call it delay, then?

Reply:
Right! we can do this to keep this side of the code clean/comprehensible. 

```go
type arg struct {
		Delay int `json:"timeout"`
}
```